package.json

```diff
@@ -47,7 +47,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --watchAll=false",
```

Comment:
is this the command that ensures that tests only run if  relevant file was changed? i don't think it's worth running `npm run test` since we do not write tests other than e2e right now. we can always turn on this feature when there are tests to run.

Reply:
Theres an additional flag that i didnt enable and that fair ill disable it for now.

Reply:
@tgrander  just disabled the pre push hooks so no tests will run on push

Reply:
awesome! will approve once tests are passing